packages/cli/src/scaling/__tests__/scaling.service.test.ts

```diff
@@ -0,0 +1,259 @@
+import { mock } from 'jest-mock-extended';
+import { ScalingService } from '../scaling.service';
+import { JOB_TYPE_NAME, QUEUE_NAME } from '../constants';
```

Comment:
Do we have some consensus if we want to rather use `@` or relative imports? I guess something like `@` by default, `./` if it's related to the same file and `../` only in tests when referring the component under test? I'm fine with either here, just wondering

Reply:
No consensus on this as far as I'm aware - I like your suggestion, ideally when we settle on a convention we can enforce it by linting so we don't have to think about it.